src/test/java/org/sonar/plugins/jacoco/JacocoSensorTest.java

```diff
@@ -176,15 +191,45 @@ void import_failure_do_not_fail_analysis() throws URISyntaxException, IOExceptio
       .build();
     tester.fileSystem().add(inputFile);
     Path sample = load("invalid_line_number.xml");
-    Files.copy(sample, temp.getRoot().toPath().resolve("invalid_line_number.xml"));
+    Path invalidFile = temp.getRoot().toPath().resolve("invalid_line_number.xml");
+    Files.copy(sample, invalidFile);
 
     sensor.execute(tester);
 
+    assertThat(logTester.logs(LoggerLevel.INFO)).contains(String.format("Importing 1 report(s): %s", invalidFile.toString()));
+
     String expectedLogError = String.format(
       "Cannot import coverage information for file '%s', coverage data is invalid. Error: java.lang.IllegalStateException: Line 1001 is out of range in the file %s (lines: 1000)",
       inputFile,
       inputFile);
-    assertThat(logTester.logs()).contains(expectedLogError);
+    assertThat(logTester.logs(LoggerLevel.ERROR)).contains(expectedLogError);
+  }
+
+  @Test
+  void should_only_log_10_but_try_to_import_all() {
+    ReportPathsProvider reportPathsProvider = mock(ReportPathsProvider.class);
+    FileLocator locator = mock(FileLocator.class);
+    ReportImporter importer = mock(ReportImporter.class);
+
+    when(reportPathsProvider.getPaths()).thenReturn(Arrays.asList(
+      Paths.get("invalid_1.xml"),
+      Paths.get("invalid_2.xml"),
+      Paths.get("invalid_3.xml"),
+      Paths.get("invalid_4.xml"),
+      Paths.get("invalid_5.xml"),
+      Paths.get("invalid_6.xml"),
+      Paths.get("invalid_7.xml"),
+      Paths.get("invalid_8.xml"),
+      Paths.get("invalid_9.xml"),
+      Paths.get("invalid_910.xml"),
+      Paths.get("invalid_911.xml")
+    ));
+
+    sensor.importReports(reportPathsProvider, locator, importer);
+    assertThat(logTester.logs(LoggerLevel.INFO)).contains(
+      "Importing 11 report(s): invalid_1.xml;invalid_2.xml;invalid_3.xml;invalid_4.xml;invalid_5.xml;invalid_6.xml;invalid_7.xml;invalid_8.xml;invalid_9.xml;invalid_910.xml;... ");
```

Comment:
If listed, I think I would prefer to have them listed in different lines.

Reply:
As discussed previously, it's better to not display the list at all.